package.json

```diff
@@ -19,7 +20,9 @@
     "ci": "npm run test-coverage && codecov < coverage/coverage-remapped.lcov",
     "build": "git rev-parse HEAD > .commithash && rollup -c && chmod a+x bin/rollup",
     "watch": "rollup -cw",
-    "prepublish": "npm run lint && npm test && npm run test:leak",
+    "prepublish": "test $(npm -v | tr . '\\n' | head -n 1) -ge '4' || exit 1",
```

Comment:
because `prepare` is npm@4 thing, im guarding here against publishing with lower versions which could potentially lead to broken published version, as fresh `build` wouldnt get executed